src/plugins/data/public/index.ts

```diff
@@ -149,11 +238,27 @@ export {
 export * from './field_formats';
 export * from './index_patterns';
 export * from './search';
-export * from './query';
+export {
+  createSavedQueryService,
+  syncAppFilters,
+  syncQuery,
+  getTime,
+  getQueryLog,
+  getQueryStateContainer,
```

Comment:
Should these be put under a namespace of some sort?

Reply:
Will get there! :)